pyproject.toml

```diff
@@ -25,7 +25,7 @@ dependencies = [
   "click~=8.1",
   "graphql-core>=3.2.0,<3.3",
   "toml~=0.10",
-  "httpx~=0.23",
+  "httpx<=0.27.0",
```

Comment:
Pinned due to `app` being [deprecated in 0.28.0](https://github.com/encode/httpx/blob/master/CHANGELOG.md#0280-28th-november-2024)